src/molecules/attendance-status.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import styles from './attendance-status.module.css';
+
+export type AttendanceStatusProps = {
+  onTime: number;
+  late: number;
+  miss: number;
+};
+
+const AttendanceStatus = ({ onTime, late, miss }: AttendanceStatusProps) => {
+  return (
+    <div lassName={styles.attendanceStatus}>
+      <div className={styles.statusItem}>出席: {onTime} 回</div>
+      <div className={styles.statusItem}>遅刻: {late} 回</div>
+      <div className={styles.statusItem}>欠席: {miss} 回</div>
+    </div>
+  );
+};
```

Comment:
```suggestion
const AttendanceStatus = ({ onTime, late, miss }: AttendanceStatusProps) => (
  <div lassName={styles.attendanceStatus}>
    <div className={styles.statusItem}>出席: {onTime} 回</div>
    <div className={styles.statusItem}>遅刻: {late} 回</div>
    <div className={styles.statusItem}>欠席: {miss} 回</div>
  </div>
);
```
アロー関数なので短くこう書いてもよいかと思います